demo/idxd-init.sh

```diff
@@ -2,8 +2,22 @@
 
 set -euo pipefail
 
-DEV="${IDXD_DEVICE_TYPE:-dsa}"
-NODE_NAME="${NODE_NAME:-}"
+DEVICE_TYPE="${DEVICE_TYPE:-dsa}"
+
+case $DEVICE_TYPE in
+    dsa)
+	IDXD_DEVICE=$DEVICE_TYPE
+        ;;
+    iaa)
+	IDXD_DEVICE=iax
```

Comment:
can this be avoided by using a newer `accel-config`?

Reply:
I have looked at the latest stable, it's still iax